### PR TITLE
feature: console integration saga-statemachine-designer

### LINKED ISSUE #5345: Integrate Saga state machine designer into the seata console
## Why you need it?
 Is your feature request related to a problem? Please describe in details

1. Integrate Saga state machine designer into the seata  console
2. Package state machine front end can automatically update the files associated with the console

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

### PR DESCRIPTION
**Pull request by liuqiufeng**
- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅱ. Does this pull request fix one issue?

fixes #5345

### CODE REVIEW

**Review by slievrly — COMMENTED** [review]
The main.js whether can be copy  in the packaging stage

File: console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
....totalMatrix=null,this._cfg.region=null},clearBBox:function(){this._cfg.box=null,this._cfg.region=null},getBBox:function(){var t=this._cfg.box;return t||((t=this.calculateBox())&&(t.x=t.minX,t.y=t.minY,t.width=t.maxX-t.minX,t.height=t.maxY-t.minY),this._cfg.box=t),t},clone:function(){var t=null,e=this._attrs,n={};return r.each(e,(function(t,i){c[i]&&r.isArray(e[i])?n[i]=function(t){for(var e=[],n=0;n<t.length;n++)r.isArray(t[n])?e.push([].concat(t[n])):e.push(t[n]);return e}(e[i]):n[i]=e[i]})),(t=new this.constructor({attrs:n}))._cfg.zIndex=this._cfg.zIndex,t}}),t.exports=s},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0}),e.setMatrixArrayType=function(t){e.ARRAY_TYPE=t},e.toRadian=function(t){return t*i},e.equals=function(t,e){return Math.abs(t-e)<=r*Math.max(1,Math.abs(t),Math.abs(e))};var r=e.EPSILON=1e-6;e.ARRAY_TYPE="undefined"!=typeof Float32Array?Float32Array:Array,e.RANDOM=Math.random;var i=Math.PI/180},,,,,,,,,,,,,,,,,,,,,,,,,function(t,e,n){var r=n(115);t.exports=function(t){return r(t,"Function")}},,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,function(t,e){var n={}.toString;t.exports=function(t,e){return n.call(t)==="[object "+e+"]"}},function(t,e,n){var r;
```
Comment (slievrly) [inline console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: `bundle.js` I think is the product after `npm build`, which is packaged into the dict directory

File: console/src/main/resources/static/console-fe/src/router.tsx [hunk 2]
```diff
@@ -21,6 +21,6 @@ import GlobalLockInfo from './pages/GlobalLockInfo';
 export default [
   // { path: '/', exact: true, render: () => <Redirect to="/Overview" /> },
   // { path: '/Overview', component: Overview },
-  { path: '/TransactionInfo', component: TransactionInfo },
-  { path: '/GlobalLockInfo', component: GlobalLockInfo },
+  { path: '/transactioninfo', component: TransactionInfo },
```
Comment (slievrly) [inline console/src/main/resources/static/console-fe/src/router.tsx:24]: restful urls are preferred.

File: console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js (hunk 1 above)
Comment (liuqiufeng) [reply console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: bundle.js是designer打包之后生成的，使用designer需要它。目前是保持目录结构与designer打包后一致，避免引用不到。有什么好的建议吗？我没太看明白意思。

File: console/src/main/resources/static/console-fe/src/router.tsx (hunk 2 above)
Comment (liuqiufeng) [reply console/src/main/resources/static/console-fe/src/router.tsx]: done.

File: console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js (hunk 1 above)
Comment (liuqiufeng) [reply console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: 是想把js放到/js文件夹下吗？如果是这样，那就需要调整下designer的代码。

Comment (slievrly) [reply console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: copy  to:
1. src/main/resources/static/console-fe/dist
2. console/target/

Comment (slievrly) [reply console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: It would be a good idea for npm to work with the maven lifecycle.

Comment (slievrly) [reply console/src/main/resources/static/saga-statemachine-designer/dist/bundle.js:1]: The exec-maven-plugin we investigated so far works, but we can also use others.

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**